ci-operator/jobs/jaegertracing/jaeger-operator/jaegertracing-jaeger-operator-main-periodics.yaml

```
@@ -14,9 +14,20 @@ periodics:
     ci-operator.openshift.io/cloud-cluster-profile: aws-cspi-qe
     ci-operator.openshift.io/variant: jaeger-product-hypershift-4.13-lp-rosa-hypershift
     ci.openshift.io/generator: prowgen
-    job-release: "4.13"
+    job-release: "4.14"
```

```suggestion
    job-release: "4.13"
```

This job is meant to test rosa-hypershift which is specific to 4.13 for now, this change on line 17 isn't needed.